README.md

```diff
@@ -22,6 +22,8 @@ It is possible to only include data in reports based on specific conditions (Pup
 
 Enabling this module is as simple as classifying your Puppet Servers with spluk_hec and setting the Splunk HEC URL along with the token provided by Splunk. This module sends data to Splunk by modifying your report processor settings and indirector routes.yaml.
 
+To send Orchestrator jobs and Event activity to Splunk, follow the instructions in the [Events](#events) Section. 
+
 There are two Tasks included in this module, `splunk_hec:bolt_apply` and `splunk_hec:bolt_result`, that provide similar data for Bolt Plans to submit data to Splunk. Example plans are included which demonstrate task usage.
```

Comment:
I'm not sure that `provide similar data for Bolt Plans to submit data to Splunk` makes it clear that these are plans that allow you to send reports to splunk as a part of a bolt plan. Try maybe `provide a pre-packaged way to compose Bolt Plans that submit data to Splunk every time they are run.`